packages/dx-chart-core/src/utils/event-tracker.test.js

```diff
@@ -171,16 +171,16 @@ describe('EventTracker', () => {
       });
 
       expect(handlers).toEqual({
```

Comment:
In that test we check mouse move handlers, thus the name of the test should not be `...pointer move handlers`.

Reply:
Removed *pointer* occurrences.